server/api/routers/ml/router.py

```diff
@@ -0,0 +1,409 @@
+"""
+ML Router - Endpoints for ML model training and inference.
+
+Provides access to:
+- Custom Text Classification (SetFit few-shot learning)
+- Anomaly Detection (train and detect anomalies)
+
+Note: OCR and Document extraction have moved to /v1/vision/*
+"""
+
+import logging
+from typing import Any
+
+from fastapi import APIRouter
+from server.services.ml_model_service import MLModelService
+from server.services.universal_runtime_service import UniversalRuntimeService
+
+from .types import (
+    AnomalyFitRequest,
+    AnomalyLoadRequest,
+    AnomalySaveRequest,
+    AnomalyScoreRequest,
+    ClassifierFitRequest,
+    ClassifierLoadRequest,
+    ClassifierPredictRequest,
+    ClassifierSaveRequest,
+)
+
+logger = logging.getLogger(__name__)
+
+router = APIRouter(prefix="/ml", tags=["ml"])
+
+
+# =============================================================================
+# Health Check
+# =============================================================================
+
+
+@router.get("/health")
+async def health_check() -> dict[str, Any]:
```

Comment:
The `/health` endpoint on the server itself should be enough to establish the current health of the runtime